open-api/rest-catalog-open-api.yaml

```diff
@@ -2731,6 +2967,32 @@ components:
           additionalProperties:
             type: string
 
+    PrePlanTableResult:
+      description:
+        When doing prePlan the result is a list of plan-tasks.
+      type: object
+      required:
+        - plan-tasks
+      properties:
+        expiration-time-ms:
+          type: integer
+          format: int64
+        plan-tasks:
+          type: array
+          items:
+            $ref: '#/components/schemas/PlanTask'
+
+    PlanTableResult:
+      description: Result of getting file-scan-tasks from a plan operation.
```

Comment:
similar comment on description, just "Result of planning a table" should be fine